### PR TITLE
Add search to popup context menu

### PR DESCRIPTION
**Pull request by agrogers**
Performs a 'phrase' search of the highlighted text. I think it would be better if it showed the text it was searching for in the main search window but i don't know how to get the search form to show itself yet. 

image

### CODE REVIEW

**Comment by agrogers** [comment]
Closes #1237

File: app/src/main/java/net/bible/android/view/activity/page/BibleView.kt [hunk 1]
```diff
@@ -203,9 +212,10 @@ class BibleView(val mainBibleActivity: MainBibleActivity,
     private var minChapter = -1
     private var maxChapter = -1
 
-
     private var gestureDetector: GestureDetectorCompat
 
+    @Inject public lateinit var documentControl: DocumentControl
```
Comment (tuomas2) [inline app/src/main/java/net/bible/android/view/activity/page/BibleView.kt]: looks unused

File: app/src/main/java/net/bible/android/view/activity/page/BibleView.kt [hunk 2]
```diff
@@ -1585,5 +1620,11 @@ class BibleView(val mainBibleActivity: MainBibleActivity,
         // never go to 0 because a bug in Android prevents invalidate after loadDataWithBaseURL so
         // no scrollOrJumpToVerse will occur
         private const val TOP_OF_SCREEN = 1
+        const val STD_REQUEST_CODE = 1
```
Comment (tuomas2) [inline app/src/main/java/net/bible/android/view/activity/page/BibleView.kt]: looks unused

**Comment by tuomas2** [comment]
I made couple of code improvements in edc3a9e, please check that :-)

**Comment by agrogers** [comment]
> I made couple of code improvements in [edc3a9e](https://github.com/AndBible/and-bible/commit/edc3a9e536ad8629e2540f9b8956c734a649d749), please check that :-)

I think a lot of that code that i added i copied from somewhere else in AndBible :) 
And I will try use the elvis operator more

**Pull request merged**